### ISSUE TITLE
[Docs]questions about lr and batchsize 

### ISSUE THREAD

**Issue by xiaxiangyu** [open-mmlab/mmrotate#645]
### Branch

master branch  https://mmrotate.readthedocs.io/en/latest/

### 📚 The doc issue

hello! I want to use a single GPU and set batchsize 8 to train r3det_kfiou_ln_swin_tiny_adamw_fpn_2x_dota_ms_rr_oc.py .  How should I set the lr .Should lr be 0.0001 x 4 ?

**Comment by zytx121** [comment]
Hi @xiaxiangyu 
According to the linear scaling principle, the best learning rate should be 0.0001 x 4. However, according to experience, the mAP will become worse if the batchsize is large. If you must increase the batchsize, better change the number of training epochs from 1x to 3x.
